docs/src/pages/components/steppers/CustomizedSteppers.tsx

```diff
@@ -96,7 +98,9 @@ const ColorlibConnector = styled(StepConnector)(({ theme }) => ({
   },
 }));
 
-const ColorlibStepIconRoot = styled('div')(({ theme, styleProps = {} }) => ({
+const ColorlibStepIconRoot = styled('div')<{
+  styleProps: { completed?: boolean; active?: boolean };
```

Comment:
So, we can drop the noisy `!!`?

Reply:
Good catch, yes we should. Let me update.